src/boot/is_debug.c

```diff
@@ -92,3 +102,5 @@ NORETURN void func_80002384(const char* exp, const char* file, u32 line) {
         ;
     }
 }
+
+#endif /* OOT_DEBUG */
```

Comment:
Not sure if we should be putting these comments. At least I'm not sure i like the idea of only putting it sometimes. 

I understand the rationale is that this is a longer block that is contained within the ifdef than usual. But most editors will show you what ifdef and endif belongs to if you highlight it

Reply:
Well, mine doesn't :p (VS Code with mostly default configuration). And GitHub won't show it either and I view code here a lot.

I noticed this in other parts of the codebase, and I've been putting it whenever I can't see the corresponding #if on my screen. I know screen sizes vary but I think it's a good heuristic

Reply:
I dont like the inconsistency it introduces. Some more opinions needed

Reply:
I don't think this is useful because it doesn't tell you what the matching `#if`/`#ifdef`/`#ifndef` is. The block may be under `OOT_DEBUG` as well as !`OOT_DEBUG`

The only spots I've seen this idiom useful is in deeply nested conditional directives (headers that shouldn't exist at all with how unreadable they are) which will never be our case

Reply:
Okay, I've removed it here and in the z_collision_check PR